apps/ledger-live-desktop/static/i18n/ar/app.json

```diff
@@ -1715,7 +1715,8 @@
     "swap" : {
       "notice" : {
         "default" : "تحقق من تفاصيل المبادلة على جهازك قبل إرسالها. يتم تبادل العناوين بشكل آمن حتى لا تضطر إلى التحقق منهم.",
-        "cic" : "تحقق من تفاصيل المبادلة على جهازك قبل إرسالها. CIC قد تحتاج إلى إجراء تحقق إضافي لبعض المعاملات، مما يؤدي إلى تأخير."
+        "cic" : "تحقق من تفاصيل المبادلة على جهازك قبل إرسالها. CIC قد تحتاج إلى إجراء تحقق إضافي لبعض المعاملات، مما يؤدي إلى تأخير.",
+        "changelly" : "تحقق من تفاصيل المبادلة على جهازك قبل إرسالها. Changelly قد تحتاج إلى إجراء تحقق إضافي لبعض المعاملات، مما يؤدي إلى تأخير."
```

Comment:
The reason I did it this way is because we already have the translation for CIC. A better way of doing this would be to change the cic translation to just pass in a placeholder. So it becomes:

```
        "provider": "Verify the Swap details on your device before sending it. {{provider}} may need to perform additional verification for some transactions, resulting in a delay.",
        ```
        
        What do you think @sarneijim ? Would that change need to go through the translation team?

Reply:
Changed https://github.com/LedgerHQ/ledger-live/pull/2159/commits/7bedd53123a9b79a38aad45e65821d3ced210196